packages/flutter/lib/src/painting/borders.dart

```diff
@@ -151,20 +159,29 @@ class BorderSide {
   /// A hairline black border that is not rendered.
   static const BorderSide none = BorderSide(width: 0.0, style: BorderStyle.none);
 
-  /// The direction of where the border will be drawn relative to the container.
-  final StrokeAlign strokeAlign;
+  /// The relative position of the stroke on a [BorderSide] in a [Border] or [OutlinedBorder].
+  /// When set to [StrokeAlign.inside], the stroke is drawn completely inside the widget.
```

Comment:
Needs a blank doc comment line after the first sentence (dartdoc uses that to make a summary line).